models/event.js

```diff
@@ -0,0 +1,54 @@
+const Sequelize = require('sequelize');
+const sequelize = require('../utils/db');
+
+class Event {
```

Comment:
A model usually is definition that can be instantiated for creating new entities (in this case new events). By creating a class that can be instantiated, and once you did that it lets you create (and other actions) events you're creating another controller.

If you want you can add some custom methods with both [class and instance methods](http://docs.sequelizejs.com/manual/tutorial/models-definition.html#expansion-of-models).